Automattic/jetpack
# Subscribe block: Add sync between button background color and border color

The proposed changes provide an _automatic border color change_ when the user changes the _button background color_.

Screen recording displaying the proposed changes in action:

https://user-images.githubusercontent.com/25105483/154711704-c3408e4d-f54f-4470-8ffd-06e951a45ead.mp4

#### Changes proposed in this Pull Request:

The PR introduces new `useEffect()` that automatically sets the `borderColor` to the currently set `buttonBackgroundColor`.

The `borderColor` is set automatically if and only if the `borderColor` holds the same color as the `buttonBackgroundColor` (including a case when the colors haven't been set yet; for instance when the block is added to the page). The condition also makes sure that if the user already set a custom color for their border manually, it won't get overridden.

#### Jetpack product discussion
- p1645093761706199/1645006316.158349-slack-C02TCEHP3HA

#### Does this pull request change what data or activity we track or use?
No, it does not.

#### Testing instructions:
1. Add the Subscribe block to a post or a page
2. Adjust its Button Background Color and Border Color settings
3. If both colors are the same, changing the Button Background Color should automatically change the Border Color - as can be seen in the screen recording above
